code/game/objects/items.dm

```diff
@@ -652,6 +661,12 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 	if(item_flags & SLOWS_WHILE_IN_HAND || slowdown)
 		user.update_equipment_speed_mods()
 	item_flags |= IN_INVENTORY
+	//MonkeStation Edit Start: Sounds on Equip/Pickup
```

Comment:
why is this including our edit? is this different than the port itself?

Reply:
Yes, it is different from the port itself.
I had to edit the port to make it work properly as well as keep the 'bug' from 53392